odonto/odonto_submissions/tests/test_models.py

```diff
@@ -171,9 +171,20 @@ class CompassBatchResponseParseTestCase(OpalTestCase):
         ]
     )
 
-    REJECTION_MESSAGE = " ".join(
+    MULTIPLE_SUCCESS_MESSAGES = " ".join(
         [
+            '<icset><ic schvn="1.0" synv="1" ori="A0DPB" dest="89651"',
+            'datrel="190730" tim="0203" seq="000009" xmcat="1">',
+            '<contrl schvn="1.0" ori="89651" dest="A0DPB" seq="000544" accd="1"',
+            '/>',
+            '<contrl schvn="1.0" ori="89651" dest="A0DPB" seq="000545" accd="1"',
+            '/>',
+            "</ic></icset>",
+        ]
+    )
```

Comment:
Why is it again that you hate triple quoted strings so much ? 
You can even have indentation for xml in them...

Reply:
so the _reason_ was because for 

```
    EMPTY_MESSAGE = """
        <receipt schvn="1.0" err="There are no
responses waiting for site 89651"/>
    """
```
wasn't pretty and everything else came out from that call. I'll change the to multiline strings and just leave the first one ugly.